docs/manual/tags.md

```diff
@@ -61,6 +61,7 @@ Cookie             | 1094 | string       | An opaque value for tracking packages
 Distribution       | 1010 | string       | Distribution name.
 Disttag            | 1155 | string       | Distribution acronym.
 Disturl            | 1123 | string       | Distribution specific URL of the package.
+Downloadurl        | 5101 | string       | URL to list of upstream releases in machine readable form if possible
```

Comment:
I quite dislike both the tag name and the docs. The name because it's so very ambiguous and even misleading, "downloading" is something *users* do, eg "where do I (re)download this very rpm package", when this is about upstream sources and robots. The doc because it's equally ambiguous and I wouldn't have a clue what to put in there: WTF does "list of releases in machine readable form, if possible" really mean? Is a list of releases "1.2", "1.3" or is it supposed to contain the actual source tarball or what? 

If it's about sources then lets say it both in the tag, and it's doc. I don't have good suggestions offhand, but to throw some ideas around: SourceURL would be fine but then it's easily confused with Source: and could be expected to have related semantics, so not good. ReleaseURL may be closer to describing the intent, especially when documented, but dunno.

Reply:
Yeah, the issue is we have used up all words in that area for other things already: Release, Source, ...

What about `UpstreamReleases` "Location/URL to check for newer releases from upstream. If possible one that can be checked automatically to notify the packager." ?

Reply:
UpstreamReleases is probably the best so far, simply because it's most to the actual point.
That doc gives the impression that *rpm* may notify the packager, which wont be the case. Maybe something more in the lines of "URL to check for newer releases from upstream. The purpose of this tag is to allow external software such as \<wellknownexample\> to check for newer upstream releases and notify the packager."

Reply:
Shortened the description a bit but kept the content more or less the same.

Reply:
I wonder if that tag should actually have an URL in the end. It's a bit long as it is, but it'd make the point more obvious...